services/platform/app/features/automations/triggers/components/schedule-create-dialog.tsx

```
@@ -0,0 +1,258 @@
+              className="mt-7 shrink-0"
+              aria-label={t(
+                'triggers.schedules.form.ai.generateButton' as any,
+              )}
+            >
+              {t('triggers.schedules.form.ai.generateButton' as any)}
+            </Button>
+          </div>
+          {cronDescription && (
+            <p
+              className="text-xs text-muted-foreground"
+              role="status"
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)